packages/rule-typescript-config/docs/import-helpers.md

```diff
@@ -0,0 +1,80 @@
+# `import-helpers`
+
+`typescript-config/import-helpers` checks that the property `importHelpers`
+is enabled in your TypeScript configuration file (i.e `tsconfig.json`) and
+that the [`tslib` package][tslib package] is installed.
+
+### Why is this important?
+
+By enabling the `importHelpers` compiler option of TyepScript the compiler will
+use the `tslib` package and reduce the size of the output. E.g.:
+
+```js
+var __assign = (this && this.__assign) || Object.assign || function(t) {
+    for (var s, i = 1, n = arguments.length; i < n; i++) {
+        s = arguments[i];
+        for (var p in s) if (Object.prototype.hasOwnProperty.call(s, p))
+            t[p] = s[p];
+    }
+    return t;
+};
+exports.x = {};
+exports.y = __assign({}, exports.x);
+```
+
+```js
+var tslib_1 = require("tslib");
+exports.x = {};
+exports.y = tslib_1.__assign({}, exports.x);
+```
+
+### What does the rule check?
+
+This rule checks the `compilerOptions` property `importHelpers` is enabled.
+
+#### Examples that **trigger** the rule
+
+By default TypeScript doesn't enable `importHelpers`:
+
+```json
+{
+    ...
+    "compilerOptions": {
+        "target": "es5"
+    }
+    ...
+}
+```
+
+Also setting the value to `false` will fail:
+
+```json
+{
+    ...
+    "compilerOptions": {
+        "importHelpers": false
```

Comment:
Maybe in these types of examples also add `...` to signalized there are multiple properties?